queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py

```diff
@@ -164,6 +167,49 @@ def _get_pdfs(
                     }
                 )
 
+            if legal_type == Business.LegalTypes.COOP.value:
+                # Add rules
+                rules = requests.get(
+                    f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}'
+                    '?type=certifiedRules',
+                    headers=headers
+                )
+                if rules.status_code != HTTPStatus.OK:
+                    logger.error('Failed to get certifiedRules pdf for filing: %s', filing.id)
+                    capture_message(f'Email Queue: filing id={filing.id}, error=certifiedRules generation',
+                                    level='error')
+                else:
+                    certified_rules_encoded = base64.b64encode(rules.content)
+                    pdfs.append(
+                        {
+                            'fileName': 'Certified Rules.pdf',
+                            'fileBytes': certified_rules_encoded.decode('utf-8'),
+                            'fileUrl': '',
+                            'attachOrder': '2'
```

Comment:
Not entirely sure how this works but just confirming that `attachOrder` being `2` for both rules and memorandum pdf is ok.

Reply:
Not sure either but i fixed it.